### ISSUE TITLE
Error creating openshift project through launch.openshift.io

### ISSUE THREAD

**Issue by heiko-braun** [openshiftio/launchpad-frontend#100]
![screen shot 2017-04-27 at 10 09 44](https://cloud.githubusercontent.com/assets/369380/25473943/bc3e68fc-2b31-11e7-9b9f-b0c9a38c19b1.png)
![screen shot 2017-04-27 at 10 09 28](https://cloud.githubusercontent.com/assets/369380/25473947/be103232-2b31-11e7-957e-f2f533a80531.png)

**Comment by edewit** [comment]
backend response is a 500 and frontend can't parse it as json:
```html
<html><head><title>Error</title></head><body>Internal Server Error</body></html>
```
Could somebody get the logs?

**Comment by edewit** [comment]
I think this is due to the fact that the "federated identities" haven't been setup. We might need a better way to indicate that ;)

**Comment by SlavoKrupa** [comment]
Is this related ?
https://github.com/openshiftio/appdev-planning/issues/21

**Comment by ALRubinger** [comment]
Should be fixed by openshiftio/appdev-planning#27.

But the JS errors should be tidied up here as well :)
